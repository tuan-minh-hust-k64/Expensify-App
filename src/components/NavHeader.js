import React from 'react';
import {NavLink} from 'react-router-dom';
const NavHeader = () => (
    <div>
        <h1>Expensify</h1>
        <ul>
            <li>
                <NavLink activeClassName="is-active" to="/create">create</NavLink>
            </li>
            <li>
                <NavLink activeClassName="is-active" to="/help">help</NavLink>
            </li>
            <li>
                <NavLink exact activeClassName="is-active" to="/">home</NavLink>
            </li>
        </ul>
    </div>
);
export default NavHeader;



