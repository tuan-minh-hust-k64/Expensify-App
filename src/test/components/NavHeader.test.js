import ReactShallowRenderer from 'react-test-renderer/shallow'
import React from "react";
import NavHeader from "../../components/NavHeader";
test('component navHeader', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<NavHeader />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});