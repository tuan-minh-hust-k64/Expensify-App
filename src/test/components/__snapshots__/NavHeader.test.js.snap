// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component navHeader 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <ul>
    <li>
      <NavLink
        activeClassName="is-active"
        to="/create"
      >
        create
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        to="/help"
      >
        help
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        exact={true}
        to="/"
      >
        home
      </NavLink>
    </li>
  </ul>
</div>
`;
